### ISSUE TITLE
[FAWE 1.14.4] No value for ArgAcceptingCommandFile

### ISSUE THREAD

**Issue by willies952002** [IntellectualSites/FastAsyncWorldEdit-1.13#249]
**FastAsyncWorldEdit Version Number:** 
```
FastAsyncWorldEdit version 1.14
Author: Empire92
```
-- *Dev Build: 82*

**Spigot/Paper Version Number:** 
```
This server is running PaperDragon version git-PaperDragon-"1c3c3b1f" (MC 1.14.4) (Implementing API version 1.14.4-R0.1-SNAPSHOT)
```

**Description of the problem:**
```
[20:06:32] [Server thread/INFO]: willies952002 issued server command: //stack 1 u
[20:06:32] [Craft Scheduler Thread - 1835/ERROR]: An unexpected error while handling a WorldEdit command
java.util.NoSuchElementException: No value for ArgAcceptingCommandFlag{description=TextComponentImpl{content=Source mask, children=[], style=Style{color=null, obfuscated=null, bold=null, strikethrough=null, underlined=null, italic=null, clickEvent=null, hoverEvent=null, insertion=null}}, name=m, argumentName=TranslatableComponentImpl{key=sourceMask, args=[], children=[], style=Style{color=null, obfuscated=null, bold=null, strikethrough=null, underlined=null, italic=null, clickEvent=null, hoverEvent=null, insertion=null}}, types=[Key{type=com.sk89q.worldedit.function.mask.Mask, annotationWrapper=NullAnnotationWrapper}], defaults=[]}
        at org.enginehub.piston.impl.CommandParametersImpl.valueOf(CommandParametersImpl.java:92) ~[?:?]
        at org.enginehub.piston.part.ArgAcceptingCommandPart.value(ArgAcceptingCommandPart.java:31) ~[?:?]
        at com.sk89q.worldedit.command.RegionCommandsRegistration.extract$sourceMask(RegionCommandsRegistration.java:1138) ~[?:?]
        at com.sk89q.worldedit.command.RegionCommandsRegistration.cmd$_stack(RegionCommandsRegistration.java:917) ~[?:?]
        at org.enginehub.piston.CommandManager.execute(CommandManager.java:158) ~[?:?]
        at com.sk89q.worldedit.extension.platform.PlatformCommandManager.lambda$handleCommandOnCurrentThread$21(PlatformCommandManager.java:682) ~[?:?]
        at com.sk89q.worldedit.extension.platform.PlatformCommandManager.handleCommandTask(PlatformCommandManager.java:698) ~[?:?]
        at com.sk89q.worldedit.extension.platform.PlatformCommandManager.handleCommandOnCurrentThread(PlatformCommandManager.java:684) ~[?:?]
        at com.sk89q.worldedit.extension.platform.PlatformCommandManager.lambda$handleCommand$19(PlatformCommandManager.java:654) ~[?:?]
        at org.bukkit.craftbukkit.v1_14_R1.scheduler.CraftTask.run(CraftTask.java:84) ~[paperdragon-1.14.4.jar:git-PaperDragon-"1c3c3b1f"]
        at org.bukkit.craftbukkit.v1_14_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:54) ~[paperdragon-1.14.4.jar:git-PaperDragon-"1c3c3b1f"]
        at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22) ~[paperdragon-1.14.4.jar:git-PaperDragon-"1c3c3b1f"]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_161]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_161]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_161]
```

**How to replicate**:
1. Select area
2. `/stack 1 u`

**Checklist**:
- [X] I included all information required in the sections above
- [X] I made sure there are no duplicates of this report [(Use Search)](https://github.com/IntellectualSites/FastAsyncWorldEdit-1.13/issues?utf8=%E2%9C%93&q=is%3Aissue)
- [X] ~I made sure I am using an up-to-date version of [FastAsyncWorldEdit for 1.13.2](https://ci.athion.net/job/FastAsyncWorldEdit-Breaking/)~
- [X] I made sure the bug/error is not caused by any other plugin

**Comment by MattBDev** [comment]
Thanks for the report! Issues like these are exactly what we have been trying find and fix.

**Comment by N0tMyFaultOG** [comment]
Wasn't that already fixed in 580fb11 ?
